frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputForm.tsx

```diff
@@ -71,53 +59,46 @@ function ActionParametersInputForm({
       .concat(hiddenFieldIds);
   }, [mappedParameters, action.visualization_settings?.fields]);
 
-  const fetchInitialValues = useCallback(async () => {
-    const prefetchEndpoint =
-      getDashboardType(dashboard?.id) === "public"
-        ? PublicApi.prefetchValues
-        : ActionsApi.prefetchValues;
-
-    const fetchedValues = await prefetchEndpoint({
-      dashboardId: dashboard?.id,
-      dashcardId: dashcard?.id,
-      parameters: JSON.stringify(dashcardParamValues),
-    }).catch(_.noop);
+  const prefetchValues = useCallback(async () => {
+    if (!fetchInitialValues) {
+      return;
+    }
 
-    if (fetchedValues) {
+    try {
+      const fetchedValues = await fetchInitialValues();
       setPrefetchedValues(fetchedValues);
+    } catch (error) {
+      // ignore silently
```

Comment:
it would be nice to have it `console.error`ed IMO

Reply:
Will it cause the error to be logged somewhere in our systems?

I don't really like to console errors out - I'm a user of some banking systems that spill lots of errors into JS console and it gives me the shivers every time I see it.

BTW this is existing code, just refactored to async/await + try/catch. The error is currently [ignored silently in `master`.](https://github.com/metabase/metabase/blob/73966fb/frontend/src/metabase/actions/containers/ActionParametersInputForm/ActionParametersInputForm.tsx#L84)

Reply:
I totally understand it, @ranquild what's our policy regarding logging errors?

Reply:
I don't know if such a policy exists; however browser logs are usually attached to issues in GitHub

Reply:
> however browser logs are usually attached to issues in GitHub

It makes sense to add it then. Done: 7b2507fc2d09ca8ef55e47d82a050d18fcd98256